### PR TITLE
chore(streaming): Better error management 

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Chore**

### Description

Trying to debug observed latency issues in prod. This adds some better error handling, so if there's an issue we still close the stream.

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -30,6 +31,10 @@ const { routes, routePaths } = getRoutes()
 app.get(
   routePaths,
   async (req: ArtsyRequest, res: ArtsyResponse, next: NextFunction) => {
+    if (ENABLE_SSR_STREAMING) {
+      res.write("<html>")
```
Comment (damassi) [inline src/server.ts:35]: Send a response immediately to queue up streaming and lower TTFB

Comment (mzikherman) [inline src/server.ts:35]: sneaky!

Comment (damassi) [reply src/server.ts]: ran into some error-page edge-cases with this :/ need to experiment more.

**Comment by damassi** [comment]
Too many edge-cases around stream errors. Closing this for now and going to pause until React 18 upgrade.

**Pull request closed without merging**